src/chains/base.ts

```diff
@@ -111,6 +111,7 @@ export enum ChainId {
   BAS = 8453,
   SCL = 534352,
   MOD = 34443,
+  MNT = 5000,
```

Comment:
I think we also need to add chainKey for the new chain

Reply:
Thanks, forgot this part, damn